dotcom-rendering/src/components/Analytics.amp.tsx

```diff
@@ -12,6 +12,7 @@ type Props = {
 export const Analytics = ({
 	analytics: { comscoreID, section = '', neilsenAPIID, ipsosSectionName },
 }: Props) => {
+	console.log(`section: ${section}`);
```

Comment:
Is this `console.log` is suppose to be just for development or you are intending to have it in the PR?

Reply:
nope just for testing - removal needed